onnx/test/version_converter_test.py

```diff
@@ -315,6 +315,25 @@ def test_reshape_4_6(self):  # type: () -> None
         assert converted_model.graph.node[0].op_type == "Reshape"
         assert converted_model.opset_import[0].version == 6
 
+    # Test Sum Adapter: 7 -> 8
+    def test_sum_7_8(self):  # type: () -> None
+        nodes = [helper.make_node('Sum', ["data_0", "data_1", "data_2",
+            "data_3", "data_4"], ["sum"])]
+        graph = helper.make_graph(
```

Comment:
Minor: Maybe add a blank line to have consistent formatting as other test points.